actionpack/test/controller/params_wrapper_test.rb

```diff
@@ -212,6 +212,16 @@ def test_empty_parameter_set
       )
     end
   end
+
+  def test_handles_empty_content_type
+    with_default_wrapper_options do
+      @request.env["CONTENT_TYPE"] = ''
```

Comment:
Should not this test setting it to `nil`? Will not the new implementation of `has_content_type?` return `true` when it is a empty string?